input/templates/mix-match.hbs

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  {{> common/htmlhead}}
+</head>
+
+<body>
+  <div class="darkbg hidden"></div>
+  {{> common/header}}
+
+  <div class="mix-match">
+    <div class="container">
+      {{> catalog/breadcrumb breadcrumb=content.breadcrumb}}
+      <div class="row">
+        <div class="col-md-4 col-md-offset-1 col-sm-6">
+          {{> catalog/product-gallery gallery=content.gallery}}
+          <div class="mix-match-view-more hidden-xs">
+            <div class="mix-match-view-more-title">{{i18n "catalog:viewMore.title"}}</div>
+            <ul class="mix-match-view-more-list list-inline">
+              {{#each content.viewMore}}
```

Comment:
I will move this also to a partial, as it is something that people would like to disable (and the easiest way to disable such a small feature is just overriding the template with an empty template :))